packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -674,3 +676,37 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world',
+        'my-app-custom-type': JSON.stringify({message: 'hello world'})
+      }];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+export const DropZoneExample = () => (
+  <div>
+    <Draggable />
+    <DropZone 
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}
+      style={{border: '2px solid white', margin: '20px', padding: '20px'}}>
```

Comment:
We could add some styles to `/example/index.css`, including for the different states.